keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -1505,6 +1529,38 @@ admission support.
 More general types of validations like this would benefit from CEL support for
 functions like `labelSelector.match()`.
 
+#### Use Case: Fine grained control of enforcement
+
+Policy author wishes to define a policy where the cluster administrator is able
+to configure if the policy is enforced by a deny or if it is only ever used to
+emit a warning.
```

Comment:
The yaml below doesn't appear to ever emit a warning?

Reply:
Oops, changed my mind on what example to use and forgot to update the intro text. Fixed.